### PR TITLE
WIP: remove MXE dependency for cross-compilation

### PR DESCRIPTION
**Pull request by kendase3**
I'm not sure if the resultant builds are statically linked yet; I'll have access to a windows machine tomorrow.  Also note that DCMAKE_C_IMPLICIT_INCLUDE_DIRECTORIES should only be set for the windows builds; right now it's unconditional.

### CODE REVIEW

File: test/slow_test.cc [hunk 1]
```diff
@@ -21,10 +21,6 @@
 #include "src/gondarwizard.h"
 #include "src/log.h"
 
-#if defined(Q_OS_WIN)
-Q_IMPORT_PLUGIN(QWindowsIntegrationPlugin);
```
Comment (nicholasbishop) [inline test/slow_test.cc]: Why remove this?

Comment (kendase3) [reply test/slow_test.cc]: it seemed as though they reworked their platform lib; my suspicion is it is no longer necessary
i'll find out tomorrow

Comment (kendase3) [reply test/slow_test.cc]: it's also possible that the fedora version of mingw qt is missing this plugin

Comment (kendase3) [reply test/slow_test.cc]: i ran into stuff basically like https://github.com/mxe/mxe/issues/1642#issuecomment-404491987

Comment (kendase3) [reply test/slow_test.cc]: here's where it's suggested linking qwindows should suffice for static windows integration goodies https://bugzilla.redhat.com/show_bug.cgi?id=1529178

**Comment by kendase3** [comment]
i'll put up an updated, more functional version of this idea.  this version is missing some goodies.

**Pull request closed without merging**